core/src/main/java/nl/nn/adapterframework/filesystem/FileSystemListener.java

```diff
@@ -150,12 +151,21 @@ public void close() throws ListenerException {
 
 	@Override
 	public Map<String,Object> openThread() throws ListenerException {
```

Comment:
Waarom kan een openThread een Map teruggeven? wat zou hier in staan?

Reply:
/**
	 * Prepares a thread for receiving messages.
	 * Called once for each thread that will listen for messages.
	 * @return the threadContext for this thread. The threadContext is a Map in which
	 * thread-specific data can be stored. 
	 */